From 3ee90909fa92feaf3df0964eed2b95c5d968ba98 Mon Sep 17 00:00:00 2001
From: Tomas Bzatek <tbzatek@redhat.com>
Date: Tue, 10 Nov 2009 14:20:18 +0000
Subject: Unfreeze updates in directory list view after rename cancellation

We successfully cancelled rename operation but the directory view
was left in frozen state, containing pseudo items with NULL elements.

Fixes bug 590591.
---
diff --git a/src/file-manager/fm-list-view.c b/src/file-manager/fm-list-view.c
index c20315d..a093d1e 100644
--- a/src/file-manager/fm-list-view.c
+++ b/src/file-manager/fm-list-view.c
@@ -1134,6 +1134,7 @@ cell_renderer_edited (GtkCellRendererText *cell,
 		g_object_set (G_OBJECT (view->details->file_name_cell),
 			      "editable", FALSE,
 			      NULL);
+		fm_directory_view_unfreeze_updates (FM_DIRECTORY_VIEW (view));
 		return;
 	}
 	
--
cgit v0.8.2
